deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+          verbs:
+          - create
+          - get
+          - list
+          - update
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes/custom-host
+          verbs:
+          - create
+        - apiGroups:
+          - workspace.devfile.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: devworkspace-controller-serviceaccount
+      deployments:
+      - name: devworkspace-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app.kubernetes.io/name: devworkspace-controller
+              app.kubernetes.io/part-of: devworkspace-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app.kubernetes.io/name: devworkspace-controller
+                app.kubernetes.io/part-of: devworkspace-operator
+            spec:
+              containers:
+              - args:
+                - /usr/local/bin/devworkspace-controller
+                - --enable-leader-election
+                env:
+                - name: DEVELOPMENT_MODE
+                  value: "true"
+                - name: RELATED_IMAGE_devworkspace_webhook_server
+                  value: quay.io/devfile/devworkspace-controller:next
+                - name: RELATED_IMAGE_project_clone
+                  value: quay.io/devfile/project-clone:next
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: devworkspace-operator
+                - name: CONTROLLER_SERVICE_ACCOUNT_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: spec.serviceAccountName
+                - name: MAX_CONCURRENT_RECONCILES
+                  value: "1"
+                - name: WEBHOOKS_SERVER_MEMORY_LIMIT
+                  value: 300Mi
+                - name: WEBHOOKS_SERVER_MEMORY_REQUEST
+                  value: 20Mi
+                - name: WEBHOOKS_SERVER_CPU_LIMIT
+                  value: 200m
+                - name: WEBHOOKS_SERVER_CPU_REQUEST
+                  value: 100m
+                - name: RELATED_IMAGE_plugin_redhat_developer_web_terminal_4_5_0
+                  value: quay.io/eclipse/che-machine-exec:nightly
+                - name: RELATED_IMAGE_web_terminal_tooling
+                  value: quay.io/wto/web-terminal-tooling:latest
+                - name: RELATED_IMAGE_pvc_cleanup_job
+                  value: quay.io/libpod/busybox:1.30.1
```

Comment:
Is there any productized busybox image that we can steal? If not I may have to work around this somehow in cpaas

Reply:
Since it's just doing `rm -rf` IIRC, we could just use `ubi8-minimal` or whatever Che does.